docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Host and deploy Blazor
+author: guardrex
+description: Discover how to host and deploy Blazor apps using ASP.NET Core, Content Delivery Networks (CDN), file servers, and GitHub Pages.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/host-and-deploy/index
+---
+# Host and deploy Blazor
+
+By [Luke Latham](https://github.com/guardrex) and [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+## Publishing
```

Comment:
The structure of this doc might be a bit confusing. I think we should make clear the two different hosting models (standalone and hosted-on-ASP.NET) and explain what you need to do to deploy in each case. Example:

This "Publishing" subheading relates to standalone but not to the hosted-on-ASP.NET model. We don't want to confuse people doing hosted-on-ASP.NET by implying they should be running `dotnet publish` on the client app.

The "Rewrite URLs for correct routing" content applies to standalone publishing only.

People using the "Blazor hosted" template (i.e., the hosted-on-ASP.NET publishing model) don't need to do anything other than the normal ASP.NET server-side publishing process, so I hope we can just link to those docs. We just need to emphasise that it's the *server* project you need to publish, not the client one (which automatically gets included with the server).